subscription-manager.spec

```diff
@@ -177,8 +177,10 @@ Requires: %{name} = %{version}-%{release}
 %if %use_gtk3
 Requires: pygobject3
 Requires: gtk3
+Requires: abattis-cantarell-fonts
 %else
 Requires: pygtk2 pygtk2-libglade
+Requires: dejavu-sans-fonts
```

Comment:
There's no way to not break SLES11 compatibility without some ugly conditionals, so I guess we may as well go ahead and handle that here (this should work):

```specfile
%if 0%{?suse_version}
Requires: dejavu
%else
Requires: dejavu-sans-fonts
%endif
```

For no particular reason, I didn't use the `font(dejavusans)` syntax for GTK2 deps suggestion above. Feel free to do that if you prefer it to be consistent.